app/controllers/api/v1/accounts/chat_status_items_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Api::V1::Accounts::ChatStatusItemsController < Api::V1::Accounts::BaseController
```

Comment:
Can we go with conversation statuses as the naming convention?  Since we don't have any objects called chats in the codebase.

Reply:
@sojan-official Done refactoring, changed chat_status_item to conversation_status